eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala

```diff
@@ -147,6 +147,10 @@ object NodeParams {
       case (old, new_) => require(!config.hasPath(old), s"configuration key '$old' has been replaced by '$new_'")
     }
 
+    // since v0.3.5 features cannot be a byte vector (hex string)
+    val isFeatureByteVector = config.getValue("features").valueType() == ConfigValueType.STRING
+    require(!isFeatureByteVector, "configuration key 'features' have moved from bytevector to human readable (ex: 'feature-name' = optional/mandatory)")
```

Comment:
nit:

```suggestion
    require(config.getValue("features").valueType() != ConfigValueType.STRING, "configuration key 'features' have moved from bytevector to human readable (ex: 'feature-name' = optional/mandatory)")
```

Reply:
Not sure about this, isn't it too big to be inline?

Reply:
It's only the text that is long, not really the equality check, so it doesn't shock me. But if you prefer the current version don't hesitate to ignore!